README.md

```diff
@@ -530,6 +530,18 @@ Ignore /var/run when taking image snapshot. Set it to false to preserve /var/run
 
 Set this flag as `--label key=value` to set some metadata to the final image. This is equivalent as using the `LABEL` within the Dockerfile.
 
+#### --dockerlint
```

Comment:
Add docs about path to `/kaniko/.hadolint.yaml` which users could override by providing it as a volume.